common/static/common/templates/components/paging-footer.underscore

```diff
@@ -1,16 +1,15 @@
-<nav class="pagination pagination-full bottom">
-<ol>
-    <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label"><%= gettext("Previous") %></span></a></li>
-    <li class="nav-item page">
+<nav class="pagination pagination-full bottom" aria-label="Full Pagination">
```

Comment:
@dan-f,  I suggested this naming convention when I falsely assumed there was a compact navigation on this page, so I would aria-label this "Teams Pagination" which makes more sense in this context.